plugins/woocommerce/e2e/tests/merchant/order-edit.spec.js

```diff
@@ -175,6 +173,8 @@ test.describe( 'Edit order > Downloadable product permissions', () => {
 		await api.delete( `orders/${ noProductOrderId }`, { force: true } );
 	} );
 
+	// these tests aren't completely independent.  Needs some refactoring.
```

Comment:
As the comment indicates, the following tests build on one another.  I think that might be the cause of them failing in subsequent runs without the environment being reset.